src/DssSpell.sol

```diff
@@ -19,28 +19,38 @@ pragma solidity 0.8.16;
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
-interface GemLike {
-    function transfer(address, uint256) external returns (bool);
-    function allowance(address, address) external view returns (uint256);
-    function approve(address, uint256) external returns (bool);
+interface VatLike {
+    function Line() external view returns (uint256);
+    function file(bytes32 what, uint256 data) external;
+    function ilks(bytes32 ilk) external view returns (uint256 Art, uint256 rate, uint256 spot, uint256 line, uint256 dust);
 }
 
-interface PauseLike {
-    function setDelay(uint256) external;
+interface VestLike {
+    function yank(uint256, uint256) external;
 }
 
-interface VestLike {
-    function restrict(uint256) external;
-    function create(address, uint256, uint256, uint256, uint256, address) external returns (uint256);
-    function yank(uint256) external;
+interface GemLike {
+    function transfer(address, uint256) external returns (bool);
 }
 
 contract DssSpellAction is DssAction {
     // Provides a descriptive tag for bot consumption
     // This should be modified weekly to provide a summary of the actions
-    // Hash: cast keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/7bedb063e5bb18d5628dcac49ee71abee7c52481/governance/votes/Executive%20vote%20-%20April%205%2C%202023.md -q -O - 2>/dev/null)"
+    // Hash: cast keccak -- "$(wget 'https://raw.githubusercontent.com/makerdao/community/d9947f3e785e1d9b1d9d24242c9e18fa268926f9/governance/votes/Executive%20vote%20-%20April%2024%2C%202023.md' -q -O - 2>/dev/null)"
```

Comment:
Nitpicking here but there are single quotes `'` inserted here that don't seem to change the output but will cascade into next spell when the crafter just changes the commit hash and date. I'm open to changing this (`wget` to `curl` as well) if you want, I'll let you do what you feel is right here since I don't mind either way. Confirmed hash is `0x6306ef3946bf3bcc63e15e5254ea6266b1c8ed54cf9d34b4f18f01d00b3be0fb` for me though, so all good.

Reply:
Regarding the quotes, I intentionally added them.

URLs may contain special shell characters and, if the crafter shell is not properly configured, they can run into problems running the command if they copy and paste it.

With single quotes we ensure that the argument is treated as a literal string, making it bulletproof, no matter what the argument is.

As a rule of thumb, always put non-trivial strings inside single quotes when dealing with shell commands.

Reply:
With the quotes is fine by me.